source/engineering/OpenFOAM/multi-node.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Multi-Node Jobs"
+---
+
+Large OpenFOAM jobs can be run on multi nodes.
+
+
+## Job Script Template
+
+Here is an example job submission script for a multi-node OpenFOAM job:
+
+```{code-block} bash
+---
+linenos: true
+---
+#!/bin/bash
+## Submission script for Cluster
+#SBATCH --job-name=my_job
+#SBATCH --time=0-01:00:00 # hh:mm:ss
+#SBATCH --nodes=4
+#SBATCH --ntasks-per-node=28
+#SBATCH --cpus-per-task=1
+
+
+#SBATCH --partition=cpu
```

Comment:
```suggestion
```

There should be no need to specify the `cpu` partition here, as this is the default partition on BC4.